### PR TITLE
Create serve configs to deploy LLMs to production

### PR DESCRIPTION
**Pull request by shrekris-anyscale**
This change adds Serve configs that can be used to deploy Aviary models to production.

### CODE REVIEW

**Review by akshay-anyscale — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -423,9 +409,9 @@ results to file.
 You can also use the Gradio API directly, by following the instructions
 provided in the [Aviary documentation](https://aviary.anyscale.com/?view=api).
 
-## Aviary Model Registry
+## RayLLM Model Registry
 
-Aviary allows you to easily add new models by adding a single configuration file.
+You can easily add new models by adding two configuration files.
 To learn more about how to customize or add new models, 
 see the [Aviary Model Registry](models/README.md).
```
Comment (akshay-anyscale) [inline README.md:416]: Replace Aviary with RayLLM?

File: README.md [hunk 2]
```diff
@@ -148,83 +78,115 @@ ray up deploy/ray/aviary-cluster.yaml
 ray attach deploy/ray/aviary-cluster.yaml
 
 # Deploy the LightGPT model. 
-aviary run --model ~/models/continuous_batching/amazon--LightGPT.yaml
+cd aviary && serve run serve/amazon--LightGPT.yaml
```
Comment (akshay-anyscale) [inline README.md:81]: why's cd aviary required?

File: README.md [hunk 3]
```diff
@@ -148,83 +78,115 @@ ray up deploy/ray/aviary-cluster.yaml
...
+pip install "aviary @ git+https://github.com/ray-project/ray-llm.git"
 ```
 
-You can query Aviary in many ways. Here we outline 4. 
+You can query your RayLLM deployment in many ways.
 
 In all cases start out by doing: 
 
 ```shell
 export AVIARY_URL="http://localhost:8000/v1"
```
Comment (akshay-anyscale) [inline README.md:103]: can we change this? It's just defining an environment variable right?

File: README.md [hunk 4]
```diff
@@ -327,107 +261,40 @@ with `python aviary/frontend/app.py`. In that case, the Gradio interface should
...
+RayLLM uses the Ray Serve CLI that allows you to interact with deployed models.
 
-Currently, `aviary` supports a few basic commands, all of which can be used with the
-`--help` flag to get more information:
 
 ```shell
-# Get a list of all available models in Aviary
-aviary models
+# Start a new model in Ray Serve from provided configuration
+cd aviary && serve run serve/<model_config_path>
```
Comment (akshay-anyscale) [inline README.md:280]: cd aviary needed?

File: README.md [hunk 5]
```diff
@@ -437,9 +304,38 @@ The easiest way is to copy the configuration of the existing model's YAML file a
...
+    model: "./models/continuous_batching/meta-llama--Llama-2-7b-chat-hf.yaml"
+```
+
+The config includes both models in the `model` argument for the `router`. Additionally, the Serve configs for both model applications are included. Save this unified config file to the `serve/` folder.
 
-Note that running `aviary run` multiple times will override the previous deployment and _NOT_ append to it.
+Run the config to deploy the models:
+
+```shell
+cd aviary && serve run serve/config.yaml
```
Comment (akshay-anyscale) [inline README.md:337]: cd aviary needed?

File: README.md (hunk 2 above)
Comment (shrekris-anyscale) [reply README.md]: `serve run` needs to be call from the repo's root directory. We used `cd aviary` to express that.

I'm not sure if it's needed though. I think using the full file path (`serve/...`) should be enough. What do you think?

File: README.md (hunk 3 above)
Comment (shrekris-anyscale) [reply README.md]: Yeah, what would you like to change it to?

File: README.md (hunk 4 above)
Comment (shrekris-anyscale) [reply README.md]: Unsure, see comment in other thread.

File: README.md (hunk 5 above)
Comment (shrekris-anyscale) [reply README.md]: Unsure, see comment in other thread.

File: README.md (hunk 3 above)
Comment (richardliaw) [reply README.md]: updated

Comment (akshay-anyscale) [reply README.md]: LLM_SERVE_URL?

**Review by akshay-anyscale — APPROVED** [review]

**Pull request merged**